packages/cli/docsgen/markdown.ts

```diff
@@ -250,7 +254,7 @@ export function renderCommandPage(
   globalOptions: CliCommandOptions<Record<never, never>>,
   lodestarCommand?: string
 ): string {
-  const page = [`# \`${cmd.command}\` CLI Command`, cmd.describe];
+  const page = [`---\ntitle: CLI Reference\n---\n# \`${cmd.command}\` CLI Command`, cmd.describe];
```

Comment:
I think we can use the frontmatter trick here too.   Just not sure if those backticks will get rendered correctly or not... 

```suggestion
  const page = [`---\ntitle: `${cmd.command}` CLI Reference\n---\n# {title}`, cmd.describe];
```

Reply:
We can just have the title set as proper `h1`, no need to duplicate. Here `title` is set as it's different from `h1` (just straight migration).